modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -126,8 +126,8 @@ def test_read_csv_empty_frame(self):
         df_equals(modin_df, pandas_df)
 
     def test_read_csv_without_glob(self):
-        with pytest.warns(UserWarning, match=r"Shell-style wildcard"):
-            with pytest.raises(FileNotFoundError):
+        with pytest.raises(FileNotFoundError):
+            with warns_that_defaulting_to_pandas():
```

Comment:
it actually raises default-to-pandas warning

Reply:
How did this work before?

Reply:
I'm not sure, it seems that it just captured every `UserWarnings` here (including default-to-pandas warning) and proceed with no errors. In the new version of pytest, the default-to-pandas warning has to be captured individually for some reason